esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -234,6 +236,15 @@ define([
                 });
                 this.wu.refresh();
             }
+            WsWorkunits.WUInfo({
```

Comment:
This specific call to WUInfo is not needed - you can just put your "then" logic within the "updateInput" callback?